src/engine/txn_engine_helper.cc

```diff
@@ -57,7 +57,7 @@ DEFINE_int64(max_resolve_count, 4096, "max rollback count");
 DEFINE_int64(max_pessimistic_count, 4096, "max pessimistic count");
 DEFINE_int64(gc_delete_batch_count, 32768, "gc delete batch count");
 
-DEFINE_bool(dingo_log_switch_txn_detail, false, "txn detail log");
+DEFINE_bool(dingo_log_switch_txn_detail, true, "txn detail log");
```

Comment:
Set this value to default false, and use gflags.conf to set this value to default true during deploy process if you want to print more logs before this version is released.